public/locales/hi/translation.json

```diff
@@ -153,7 +153,7 @@
     "postalCode": "पोस्टल कोड",
     "sortingCode": "सॉर्टिंग कोड",
     "state": "राज्य",
-    "userRegistrationRequired": "उपयोगकर्ता पंजीकरण आवश्यक",
+    "Is Public": "सार्वजनिक है",
```

Comment:
This is not in keeping with the camel case standard. Remove the space, use camel case and update the source code.